crates/ironrdp-rdpdr/src/pdu/esc.rs

```diff
@@ -5,7 +5,7 @@
 pub mod ndr;
 pub mod rpce;
 use super::efs::IoCtlCode;
-use crate::pdu::esc::ndr::{Decode, Encode};
+use crate::pdu::esc::ndr::{Decode as NdrDecode, Encode as NdrEncode};
```

Comment:
Nit: not a blocker, but it might be sensible to simply name this trait `NdrDecode` instead of using `Decode as NdrDecode`